applications/teletype/src/templates/teletype_low_balance.erl

```diff
@@ -76,7 +76,7 @@ handle_req(DataJObj) ->
     Macros = [{<<"system">>, teletype_util:system_params()}
               ,{<<"account">>, teletype_util:account_params(DataJObj)}
               ,{<<"current_balance">>, get_current_balance(DataJObj)}
-              ,{<<"threshold">>, teletype_util:get_balance_threshold(DataJObj)}
+              ,{<<"threshold">>, teletype_topup:get_balance(DataJObj)}
```

Comment:
teletype_topup:get_balance calls wht_util:current_balance which doesn't seem to be a threshold ?